packages/react-components/react-combobox/stories/src/Combobox/ComboboxCustomOptions.stories.tsx

```diff
@@ -31,9 +31,6 @@ const useStyles = makeStyles({
     gap: '2px',
     maxWidth: '400px',
   },
-  listbox: {
-    maxHeight: '200px',
```

Comment:
Why was `maxHeight` removed? As I understood from the comment in one of those issues, that for `maxHeight` to work `autoSize` is needed. Maybe it's good to add this information to the story description.

Reply:
Because it has no affect in this particular story, it was set there, but it's redundant -> it does nothing, I could add `autoSize` to make it work, but it's not needed for this story with such amount of items and where the primary purpose to show the behaviour of custom options 🙂  

![Screenshot 2024-07-24 at 16 09 21](https://github.com/user-attachments/assets/c87a3544-8f41-4e91-81bb-a301de5e4d20)

Reply:
> Maybe it's good to add this information to the story description.

Not sure about that, the requests came because users copied our examples which were incorrect `maxHeight` without `autoSize`, the bug on `Virtualizer` story is fixed and `maxHeight` was removed from `CustomOptions`, where it could bring confusion.

Tbh, do not think it should be mentioned here as it's a part of positioning API and has to be added [there](https://react.fluentui.dev/?path=/docs/concepts-developer-positioning-components--default#auto-size-for-small-viewport.

Reply:
It's true. But a lot of people don't go to positioning API. Probably it's good at least to add a note, that they should go there, because it might be not obvious for partners